webpack.config.js

```diff
@@ -26,7 +26,11 @@ module.exports = {
         ]
     },
     entry: {
-        app: './assets/js/main.js'
+        app: [
```

Comment:
Added based on https://babeljs.io/docs/en/babel-plugin-syntax-dynamic-import#working-with-webpack-and-babel-preset-env

We load a polyfill for `Promise` via polyfill.io but this seems more explicit. If this tests OK we can remove the version from polyfill.io